test/unit/test-test.coffee

```diff
@@ -43,10 +43,7 @@ describe 'Test', ->
         test.request.body =
           body: 'value'
         test.response.status = 201
-        test.response.schema = """
-          type: 'string'
-          name: 'string'
-        """
+        test.response.schema = [{ type: 'object', properties: { type: 'string', name: 'string'}}]
```

Comment:
No particular reason except that this is what we're using so it's what I'm used to.